packages/web/pages/profile/setup/timeZone.tsx

```diff
@@ -0,0 +1,38 @@
+import { SetupProfile } from 'components/Setup/SetupProfile';
+import { SetupTimeZone } from 'components/Setup/SetupTimeZone';
+import { SetupContextProvider } from 'contexts/SetupContext';
+import { useUser } from 'lib/hooks';
+import { InferGetStaticPropsType } from 'next';
+import React, { useState } from 'react';
+
+export const getStaticProps = async () => {
+  return {
+    props: {
+      hideAppDrawer: true
+    }
+  };
+};
+
+export type DefaultSetupProps = InferGetStaticPropsType<typeof getStaticProps>;
+
+const TimeZoneSetup: React.FC<DefaultSetupProps> = () => {
+
+  const [timeZone, setTimeZone] = useState<string>('');
+  const { user } = useUser({ redirectTo: '/' });
+
+  if (user?.player) {
+    const {player} = user;
+    if (player.timezone && !timeZone) {
```

Comment:
I looked into this a bit. I think the best solution would be to refactor each `<SetupComponent>` to just take a `player` as a prop and thus figure out the value of the field, rather than the parent component having this logic. Since each setup is its own page, we shouldn't need `useState` anymore either, as the data will be retrieved from graphql on load.